apps/e2e-test/features/publisher-camera-stream.feature

```diff
@@ -44,7 +44,6 @@ Feature: Publisher Camera Streaming
         And switch to "publisher-streaming" page on "publisher" app
         When the publisher clicks on the "stop button"
         Then the publisher should be navigated to "preview" page
-        And the "header" should be displayed with default values
```

Comment:
This test is no longer accurate, the new designs do not show the header in publisher app

Reply:
@vishalkharge  pls help to solve this issue.

Reply:
@jackshen you could resolve this comment after you create the issue for Vishal

Reply:
#372